migration/README.md

```diff
@@ -0,0 +1,30 @@
+# Migration
+
+Module containing [OpenRewrite](https://docs.openrewrite.org/) recipes for migrating Axon Framework applications.
+
+## Migrate to Axon Framework 4.7 Javax
+
+### Maven
+
+```bash
+mvn -U org.openrewrite.maven:rewrite-maven-plugin:run \
+  -Drewrite.recipeArtifactCoordinates=org.axonframework:axon-migration:4.7.2-SNAPSHOT \
```

Comment:
This is kind of a chicken-egg problem. But maybe we need to assume for the readme it's already been released as 4.7.2, removing `-SNAPSHOT`. Alternatively add to use the snapshot repository, so it can be used right after it's merged to master.

Reply:
This ought to work as soon as there's a release, and ensures the docs don't have to be updated
```suggestion
  -Drewrite.recipeArtifactCoordinates=org.axonframework:axon-migration:LATEST \
```

Reply:
The SNAPSHOT was mostly there in case you wanted to try it out before then; but now that you know we can lose it.